### ISSUE TITLE
Network Error: Failure To Load Api

### ISSUE THREAD

**Issue by karltaylor** [karakanb/devo#13]
```
Failed to load https://devo.burakkarakan.com/api/github: Redirect from 'https://devo.burakkarakan.com/api/github' to 'https://devo.burakkarakan.com/api/github' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'chrome-extension://elkhalpmbmbaeoemecpcfdcoekmpgmdm' is therefore not allowed access.
```

Recently been getting some errors and hackernews / github not loading:

<img width="886" alt="screen shot 2018-10-12 at 13 19 01" src="https://user-images.githubusercontent.com/6763109/46868804-795a3e00-ce21-11e8-97a9-4316a413f842.png">

**Comment by karakanb** [comment]
Thanks for reporting. Actually, I have no specific idea as Chrome extensions are not limited by CORS controls by default, [as this document says](https://developer.chrome.com/extensions/xhr), but the document also states that I need to set up a `permissions` key in the `manifest.json` file, which I didn't add. In this case, I would expect Chrome to complain for every request, but for now it seems like random to me. I will add the key and publish a new version to see whether this prevents it or not.

**Comment by karakanb** [comment]
This one is resolved via 6a6e0b8.
